olive/data/component/text_generation.py

```diff
@@ -118,7 +128,9 @@ class TextGenPairParams(TextGenParams):
 
     pair_format: TextGenPairFormat = TextGenPairFormat.DEFAULT
     input_col: str = None  # required when pair_format is CUSTOM
+    input_template: str = None  # a python f-string template for the input, input is replaced with {input}
```

Comment:
>input_template [](http://example.com/codeflow?start=4&length=14)

from semantic point of view, the input_template and input_col seems represent the same thing?

Reply:
Pushed a commit to make only one required. The priority is `template` > `column name`